src/main/java/com/sparta/popupstore/domain/review/entity/Review.java

```diff
@@ -30,13 +31,19 @@ public class Review extends BaseEntity {
     private int star;
     private String imageUrl;
 
+    private LocalDateTime createdAt;
+
+    private LocalDateTime updatedAt;
+
     @Builder
-    public Review(Long id, PopupStore popupStore, User user, String contents, int star, String imageUrl) {
+    public Review(Long id, PopupStore popupStore, User user, String contents, int star, String imageUrl, LocalDateTime createdAt, LocalDateTime updatedAt) {
```

Comment:
```suggestion
    public Review(Long id, PopupStore popupStore, User user, String contents, int star, String imageUrl) {
```
생성일과 수정일은 jpa Auditing이 자동으로 넣어주는 값입니다.